src/vs/workbench/contrib/comments/browser/commentFormActions.ts

```diff
@@ -34,16 +36,30 @@ export class CommentFormActions implements IDisposable {
 		const groups = menu.getActions({ shouldForwardArgs: true });
 		let isPrimary: boolean = !hasOnlySecondaryActions;
 		for (const group of groups) {
-			const [, actions] = group;
+			const [groupId, actions] = group;
 
 			this._actions = actions;
-			for (const action of actions) {
+			for (const current of groupId === 'inline' ? actions : [actions]) {
```

Comment:
Is the idea here that any actions with the same group would be in a dropdown? That isn't consistent with how we use groups or make dropdowns in other places. We use submenus in other places to create dropdowns.